src/ir/function.rs

```diff
@@ -387,6 +392,24 @@ impl FunctionSig {
             }
         };
 
+        let mut must_use = false;
```

Comment:
Could you move all this block to a function in `clang.rs`, probably a method of `cursor`?

Something like the following maybe:

```rust
pub fn has_simple_attr(&self, attr: &str) -> bool;
```

Then this code could just be:

```rust
let must_use = cursor.has_simple_attribute("warn_unused_result");
```

WDYT?